src/encoded/tests/features/test_news.py

```diff
@@ -0,0 +1,13 @@
+""" These scenarios do not require the workbook fixture, but get it anyway.
```

Comment:
I think the workbook feature loads all of the data inserts that these rely on?

Reply:
Please, rephrase it.